gapic-generator-java/DEVELOPMENT.md

```diff
@@ -1,210 +1,145 @@
 # Development Workflow
 
-## Set Up
-
-1.  Clone this repo.
-
-2.  Copy the Git pre-commit hooks. This will automatically check the build, run
-    tests, and perform linting before each commit. (Symlinks don't seem to work,
-    but if you find a way, please add it here!)
-
-    ```sh
-    cp .githooks/pre-commit .git/hooks/pre-commit
-    ```
-
-## Code Formatting
-
--   Run linter checks without actually doing the formatting.
-
-    ```sh
-    bazel run //:google_java_format_verification
-    ```
-    or
-    ```sh
-    mvn fmt:check
-    ```
-
--   Format files.
-
-    ```sh
-    bazel run //:google_java_format
-    ```
-    or
-    ```sh
-    mvn fmt:format
-    ```
-
 ## Test Running
 
--   Run all unit and integration tests.
+- Run all unit and integration tests.
 
     ```sh
-    bazel test //...
+    mvn install          # unit tests, maven test wouldn't work in root folder because gapic-generator-java is dependant on test jars of gax-java
+    bazel test //...  # integration tests
```

Comment:
The integration tests can only be run from the root folder, and integration tests can also used for changes in gax, so we may want to move this line and references of integration tests to root folder.

Reply:
Because this integration test is for gapic-generator-java submodule, I keep it here noting that developers should run these commands in the root directory of the repository.